ui/static/ui/js/listing.js

```diff
@@ -1,6 +1,78 @@
 define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
```

Comment:
um, I think you were replying to something different, it is still wrongly jQuery in master, and should be $ since we are using it as such everywhere